paimon-shade-caffeine-2/src/main/resources/META-INF/NOTICE

```diff
@@ -1,5 +1,5 @@
 paimon-shade-caffeine-2
-Copyright 2023-2023 The Apache Software Foundation
+Copyright 2014-2023 The Apache Software Foundation
```

Comment:
Flink Table Store was developed in 2022, I think here should be 2022-2023.

Reply:
done.